doc/htmldoc/synapses/simulations_with_gap_junctions.rst

```diff
@@ -61,25 +55,29 @@ level with e.g. the ``random`` module of the Python Standard Library:
     # total number of gap junctions
     n_gap_junction = 3000
 
+    gap_weight = 0.5
     n = nest.Create('hh_psc_alpha_gap', n_neuron)
```

Comment:
Mix of single and double quotation marks.

Reply:
all now  "